nsv13/code/controllers/subsystem/starsystem.dm

```diff
@@ -0,0 +1,206 @@
+//Subsystem to control overmap events and the greater gameworld
+SUBSYSTEM_DEF(starsystem)
+	name = "Starsystem"
+	wait = 10
+	flags = SS_NO_INIT
+	var/last_combat_enter = 0 //Last time an AI controlled ship attacked the players
+	var/modifier = 0 //Time step modifier for overmap combat
+	var/list/systems = list()
+	var/datum/starsystem/hyperspace //The transit level for ships
+	var/bounty_pool = 0 //Bounties pool to be delivered for destroying syndicate ships
+
+/datum/controller/subsystem/starsystem/fire() //Overmap combat events control system, adds weight to combat events over time spent out of combat
+	if(last_combat_enter + (5000 + (1000 * modifier)) < world.time) //Checking the last time we started combat with the current time
+		var/datum/round_event_control/lone_hunter/LH = locate(/datum/round_event_control/lone_hunter) in SSevents.control
+		var/datum/round_event_control/belt_rats/BR = locate(/datum/round_event_control/belt_rats) in SSevents.control
+		modifier += 1 //Increment time step
+		if(modifier == 13) // 30 minutes

```

Comment:
This isn't hugely clear, I would personally do something like:
next_nag_time = world.time + 30 MINUTES
if(world.time >= next_nag_time)
//code

Reply:
Incorporated.

Reply:
[https://cdn.discordapp.com/attachments/593077519247474689/683285148183166993/unknown.png](url)